feeadjuster/feeadjuster.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+import random
+import time
+from pyln.client import Plugin, Millisatoshi, RpcError
+
+
+plugin = Plugin()
+# Our amount and the total amount in each of our channel, indexed by scid
+plugin.adj_balances = {}
+# Cache to avoid loads of calls to getinfo
+plugin.our_node_id = None
+# Users can configure this
+plugin.update_treshold = 0.05
+
+
+def get_ratio(our_percentage):
+    """
+    Basic algorithm: the farther we are from the optimal case, the more we
+    bump/lower.
+    """
+    return 50**(0.5 - our_percentage)
```

Comment:
Just computed the range of these functions to make sure I understand them:

 - `x=1` -> 0.14 ~= 1/7.14
 - `x=0.5` -> 1
 - `x=0` -> 7.0.7

So the ratio moves in the range `[1/7; 7]` approximately.

Reply:
I wonder if we should normalize this, to allow the user to specify the range.

Reply:
i should add a comment with this

Re normalization: my plan was to have many such "ratio getters" with diffferent formulae that the user could activate using an --adjustment-method option (eg simple, conservative, exhaustive, linear, etc..)